css/index.css

```diff
@@ -1,3 +1,520 @@
+	z-index: 10;
+    display: flex;
+
+}
+
+header nav {
+	height: 60px;
+	width: 100%;
+	display: flex;
+	position: relative;
+	line-height: 25px;
+	flex-direction: row;
+	
+}
+
+header nav a {
+	margin: 10px 60px 0 20px;
+	text-decoration: none;
+	justify-content: flex-start;
+	color: rgb(68, 69, 160);
+	line-height: 36px;
+	font-size: 14px;
+}
+
+header .logo {
+	margin-top: 5px;
+	margin-right: 5px;
+	height: 40px;
+    width: auto;
+	justify-content: flex-start;
+}
+
+/* Hands down the most challenging part, at one point the fonts even got wacky on me */
+.main-content {
+	width: 100%;
+	height: 475px;
+	position: relative;
+	display: flex;
+	margin-top: 60px;
+}
+
+.main-content .stack-text {
+	position: absolute;
+	margin-top: 50px;
+	margin-left: 35px;
+	height: 310px;
+	width: 60%;
+	text-align: center;
+	line-height: 80px;
+	font-size: 75px;
+}
+
+.main-content .snippy {
+	position: absolute;
+	right: 0;
+	top: 10;
+	height: 380px;
+	width: 40%;
+}
+
+.main-content .start {
+	position: absolute;
+	border: 1px solid gray;
+	left: 230px;
+	bottom: 125px;
+	padding: 5px 45px 3px 45px;
+	height: 35px;
+	font-size: 18px;
+	text-align: center;
+}
+
+/* After what happened ^^^ there, this was pretty simple */
+h2 {
+	font-size: 16px;
+}
+
+.info-content .feature-about {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+.info-content .feature {
+	padding-top: 10px;
+	margin-right: 20px;
+	width: 45%;
+}
+
+.info-content .about {
+	padding-top: 10px;
+	margin-left: 20px;
+	width: 45%;
+}
+
+.middle-img {
+	margin: 35px 0;
+	width: 100%;
+}
+
+.info-content .ser-pro-vis {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+.info-content .services {
+	margin-right: 10px;
+	width: 30%;
+	padding-bottom: 25px;
+}
+
+.info-content .product {
+	margin: 0 5px;
+	width: 30%;
+}
+
+.info-content .vision {
+	margin-left: 10px;
+	width: 30%;
+}	
+
+footer {
+	text-align: center;
+	margin-top: 100px;
+}
+
+/* And so begins the tablet break */
+
+@media (max-width: 800px){
+
+    .container {
+        width: 800px;
+    }
+
+    header {
+        height: 130px;
+        width: 800px;
+        position: fixed;
+        top: 0;background-image: none;
+        z-index: 10;
+        display: flex;
+        flex-direction: column;
+    
+    }
+
+    header nav {
+        height: 60px;
+        width: 800px;
+        display: flex;
+        position: relative;
+        line-height: 25px;
+        flex-direction: row;
+        
+    }
+    
+    header nav a {
+        margin: 95px 55px 0 28px;
+        text-decoration: none;
+        justify-content: flex-start;
+        color: rgb(159, 159, 180);
+        line-height: 36px;
+        font-size: 16px;
+        font-weight: bold;
+    }
+
+    
+    header .logo {
+        margin-bottom: 65px;
+        height: 36px;
+        width: auto;
+        align-self: center;
+    }
+    
+    .main-content .snippy {
```

Comment:
I assume the snippy is the big great idea logo image by the property, not many other people would be able to pick this project to work on it and immediately know what it is. Imagine yourself coming back to it a few weeks or months from now and looking at that. You'll end up having to research it.